ci/cppcheck/cppcheck-new-issues.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-scriptdir="$(cd "${0%/*}" && pwd)"
-rootdir="${scriptdir%/*/*}"
+scriptdir=$(dirname "$(readlink -f "$0")")
```

Comment:
Which shellcheck error are you solving here? The original is the typical way of getting the current directory, why change it?

Reply:
Reverted changes.

I found why we see different number of shellcheck issues:
1) Ubuntu 18.04 uses shellcheck 0.4.6
    To install newer version (0.7.0) it's needed to use snap
    `sudo snap install shellcheck`
2) I run shellcheck without args
    Now use next command:
         `shellcheck -P SCRIPTDIR -x <script-path>`
3) I reconfigured Syntastic plugin in my Vim to see the same results.
     I've added next in my `~/.vimrc`
          `let g:syntastic_sh_shellcheck_args = "-P SCRIPTDIR -x"`